test/func/fixtures/playwright/tests/failed-describe.spec.ts

```diff
@@ -38,7 +38,7 @@ test.describe('failed describe', () => {
     test('test with successful assertView and error', async ({page, baseURL}) => {
         await page.goto(baseURL as string);
 
-        await expect(page.locator('header')).toHaveScreenshot('header-success.png');
+        await expect(page.locator('header')).toHaveScreenshot('header-success.png', {maxDiffPixelRatio: 1});
```

Comment:
As its "test with successful assertView" (fixture), we can make this test 100% stable